sparse/utils.py

```diff
@@ -240,3 +242,44 @@ def normalize_axis(axis, ndim):
         return tuple(normalize_axis(a, ndim) for a in axis)
 
     raise ValueError("axis %s not understood" % axis)
+
+
+def equivalent(x, y):
+    return (x == y) | ((x != x) & (y != y))
+
+
+def check_fill_value(nargs):
+    def generator(func):
+        @functools.wraps(func)
+        def wrapped(*args, **kwargs):
+            for arg in args[:nargs]:
+                if hasattr(arg, 'fill_value') and \
```

Comment:
Nit: per PEP 8, prefer using extra parentheses `()` to explicit continuation with `\`.